SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerAlgorithm.cc

```diff
@@ -563,14 +567,51 @@ void Phase2TrackerDigitizerAlgorithm::induce_signal(
     }
   }
   // Fill the global map with all hit pixels from this event
+  bool reweighted = false;
+  size_t ReferenceIndex4CR = 0;
```

Comment:
does it have to be upper case? It appears it does not directly contradict `cmssw` code rules, but looks unusual.

Reply:
sure I can change it